docs/faq.md

```diff
@@ -2,53 +2,113 @@
 
-You can open a bounty and if developers are interested by the job, they might do it. Read this: [Bountysource](bountysource.md)
+- Why does AzerothCore not mimic "blizzlike" bugs or exploits?
+  - Although we strive to provide blizzlike content we also value user experience. This means that we sometimes fix bugs or exploits which existed during retail at the time to provide a better overall experience for the players.
 
-#### 4 - I have an issue, where can I get help?
+- How can I contribute?
+  - You can help fixing issues by submitting Pull Requests, read more [here](contribute.md).
+  - You can help us by testing our [Pull Requests](contribute#how-to-test-a-pull-request) and participating on the [github issues' discussions](https://github.com/azerothcore/azerothcore-wotlk/issues).
+  - You can put bounties (money) on issues or suggestions that you want fixed using bountysource. Learn more here: [Bountysource](bountysource.md)
+  - You can help improve the wiki by submitting a [Pull Request](https://github.com/azerothcore/wiki).
 
-Read this carefully: [How to ask for help](how-to-ask-for-help.md).
+- Why do you not merge my fix?
+  - All fixes must be reviewed by a dev. All devs don't know everything, so you need to wait until someone reviews it.
+  - Some fixes requires testing and not all devs can test so you need to wait on someone else to test it.
+  - They get merged eventually after getting the To Be Merged label.
+  - Maybe you are not following the [SQL/C++ standards](https://www.azerothcore.org/wiki/standard-operating-procedure).
+  - Read more in [Merge Process](merge-process.md).
 
-#### 5 - Why is it better to ask on Stackoverflow for support questions?
+- How do I report a crash?
+  - By pasting your crashlog into a PasteBin or Gist.
+  - The crashlog **needs to come from a RelWithDebInfo or Debug compilation**. If it is from Release it is useless.
 
-- Because your question (and its solutions) **stays there** and you or other users **can find it later**, even using **google**
-- Because your question can easily **get lost in the chat**, and asking in the chat can only get answers by the users who are **currently online**
-- Because it's **super easy** (you can log in Stackoverflow using your **Google** or **Facebook** account - **NO NEED TO CREATE A NEW ACCOUNT**)
-- Because you can add common **tags** to your questions (like `c++`, `sql`, `docker`, `lua`, etc...) and have the chance to  **receive help from users** that you would not find in the AC discord chat
-- Because it gives extra **visibility** to our project
-- Because it helps to **avoid duplicate questions**
-- Because by posting questions or answers you can earn **points** (a profile with many points Stackoverflow can help A LOT when you want to find a job as a developer)
+- Do you support Repacks based on AzerothCore?
+  - No. Compile the code yourself if you want support.
```

Comment:
```suggestion
  - No. Repacks are NOT supported and we strongly suggest to not use them for [several reasons](https://www.mangosrumors.org/why-you-should-not-use-repacks-to-run-your-wow-server/). You can check [this tutorial](https://www.chromiecraft.com/how-to-install-a-wow-server-on-your-own-computer/) for an easy way of installing AC without using any repack.
```

Reply:
Me wanting to be a bitch to repacks users :(